dashboard-widgets.el

```diff
@@ -760,7 +778,11 @@ Argument IMAGE-PATH path to the image."
                                          (memq 'scale (funcall 'image-transforms-p)))
                                 img-props))))))
         (add-text-properties start (point) `(display ,image-spec))
-        (when (dashboard--type-is-gif-p img) (image-animate image-spec 0 t)))
+        
+(when (dashboard--is-image-animated-p img)
+  (image-animate image-spec 0 t)))
+
+      
```

Comment:
The indentation seems a bit off. Make sure there are no extra newlines. 🤔

Reply:
@jcs090218 i have made the changes.